optimum/intel/openvino/modeling_base.py

```diff
@@ -619,10 +619,8 @@ def _from_transformers(
             )
             compile_only = False
 
-        # If load_in_8bit and quantization_config not specified then ov_config is set to None and will be set by default in convert depending on the model size
-        if load_in_8bit is None and not quantization_config:
-            ov_config = None
-        else:
+        ov_config = kwargs.get("ov_export_config")
```

Comment:
can you please add test with passing ov_export_config?
Also, please update text2speech modelling file